tasks/coverage/src/suite.rs

```diff
@@ -285,6 +285,12 @@ pub trait Case: Sized + Sync + Send + UnwindSafe {
             .with_module_record_builder(true)
             .with_check_syntax_error(true)
             .build(program);
+        if !source_type.is_typescript_definition() && !source_type.is_typescript() {
+            assert!(
```

Comment:
We should return `Test::Result::ParseError("Unset symbol / reference", true)` instead of panicking here.